MvvmCross/Binding/iOS/Views/MvxBasePageViewSource.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace MvvmCross.Binding.iOS.Views
+{
+    using Platform;
+    using System;
+    using MvvmCross.Platform.Exceptions;
+    using UIKit;
+
+    public abstract class MvxBasePageViewSource : UIPageViewControllerDataSource
+    {
+
+        private readonly UIPageViewController _pageView;
+
+        protected MvxBasePageViewSource(UIPageViewController pageView)
+        {
+            this._pageView = pageView;
+        }
+
+        protected UIPageViewController PageView => this._pageView;
+
+        public virtual void ReloadData()
+        {
+            try
+            {
+                var controller = GetInitialViewController();
+                var list = new UIViewController[] { controller };
+                // FIXME: we are forced here to have animated:false because of a bug in iOS up to 9.x
```

Comment:
OK, but I have to check if this bug have been fixed in iOS 10 first. If fixed, I'll add the check.